src/combinatorics/partition.lean

```diff
@@ -0,0 +1,125 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+
+import combinatorics.composition
+import data.nat.parity
+import tactic.apply_fun
+
+/-!
+# Partitions
+
+A partition of an integer `n` is a way of writing `n` as a sum of positive integers, where the order
```

Comment:
```suggestion
A partition of a natural number `n` is a way of writing `n` as a sum of positive integers, where the order
```

Reply:
Thanks! I've made the analogous change in `composition.leam` too